### ISSUE TITLE
Spanish book selectori issue

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#2196]
![Screenshot_20220711-025715](https://user-images.githubusercontent.com/5811789/179226625-20964586-8ae3-4b48-b16d-cdbc232be499.png)

https://support.andbible.org/scp/tickets.php?id=1524

**Comment by agrogers** [comment]
I noticed this also. I can use the same logic as i used on the PR to size the text based on the number of characters. However it is not a beautiful solution because it really needs to know how wide the buttons are first. I have assumed a portrait phone layout.

**Comment by tuomas2** [comment]
this can be fixed in transifex i believe?

**Comment by timbze** [comment]
> this can be fixed in transifex i believe?

Yes, I've shortened it to "Cant" now. If some Spanish speaking person knows a better short word they can let me know

**Comment by timbze** [comment]
James (Santiago) is shortened to "Sant", so I think "Cant" is good
